src/ai/backend/agent/docker/agent.py

```diff
@@ -769,6 +770,31 @@ def protected_services(self) -> Sequence[str]:
             case ResourceGroupType.STORAGE:
                 return ("ttyd",)
 
+    async def _apply_seccomp_profile(self, container_config: MutableMapping[str, Any]) -> None:
+        default_seccomp_pth = self.resolve_krunner_filepath("runner/default-seccomp.json")
```

Comment:
Is this a typo of `path`? I have not seen any common cases of shortening `path` as `pth`...

Reply:
I will change it to a more common name,`default_seccomp_path`.